problem with GitHub Actions un-pak-ing an odbc Linux package

Looks like there's a problem with "libodbc1_2.3.11_amd64.deb".  It's unlikely related to any R code in the package.  Probably related to Linux packages beyond our control.  
- [x] I'll run it twice to rule out something flukey
- [x] I'll see if there are dependencies I can remove.  
- [x] I'm already using the latest version of r-lib's GitHub Actions [setup-r-dependencies](https://github.com/r-lib/actions/tree/v2/setup-r-dependencies)
- [ ] I'll see if I can clear/invalidate the cache through r-lib's GitHub Actions [setup-r-dependencies](https://github.com/r-lib/actions/tree/v2/setup-r-dependencies#usage)
- [ ] if these don't work, maybe give it a day for the Linux packages to work out themselves?

from https://github.com/OuhscBbmc/REDCapR/actions/runs/4178765449/jobs/7237945330#step:5:3825

```
  (Reading database ... 245555 files and directories currently installed.)
  Preparing to unpack .../libodbc1_2.3.11_amd64.deb ...
  Unpacking libodbc1:amd64 (2.3.11) ...
  dpkg: error processing archive /var/cache/apt/archives/libodbc1_2.3.11_amd64.deb (--unpack):
   trying to overwrite '/usr/lib/x86_64-linux-gnu/libodbc.so.2.0.0', which is also in package libodbc2:amd64 2.3.9-5
  dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
  Selecting previously unselected package odbcinst1debian2:amd64.
  Preparing to unpack .../odbcinst1debian2_2.3.11_amd64.deb ...
  Unpacking odbcinst1debian2:amd64 (2.3.11) ...
  dpkg: error processing archive /var/cache/apt/archives/odbcinst1debian2_2.3.11_amd64.deb (--unpack):
   trying to overwrite '/usr/lib/x86_64-linux-gnu/libodbcinst.so.2.0.0', which is also in package libodbcinst2:amd64 2.3.9-5
  dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
  Preparing to unpack .../odbcinst_2.3.11_amd64.deb ...
  Unpacking odbcinst (2.3.11) over (2.3.9-5) ...
  dpkg: error processing archive /var/cache/apt/archives/odbcinst_2.3.11_amd64.deb (--unpack):
   trying to overwrite '/etc/odbc.ini', which is also in package unixodbc-common 2.3.9-5
  Preparing to unpack .../unixodbc_2.3.11_amd64.deb ...
  Unpacking unixodbc (2.3.11) over (2.3.9-5) ...
  Preparing to unpack .../unixodbc-dev_2.3.11_amd64.deb ...
  Unpacking unixodbc-dev (2.3.11) over (2.3.9-5) ...
  Errors were encountered while processing:
   /var/cache/apt/archives/libodbc1_2.3.11_amd64.deb
   /var/cache/apt/archives/odbcinst1debian2_2.3.11_amd64.deb
   /var/cache/apt/archives/odbcinst_2.3.11_amd64.deb
  needrestart is being skipped since dpkg has failed
  E: Sub-process /usr/bin/dpkg returned an error code (1)
  Error: 
  ! error in pak subprocess
  Caused by error in `processx::run(sh, cmdline, stdout_callback = callback, stderr_to_stdout = TRUE)`:
  ! System command 'sudo' failed
  ---
  Backtrace:
  1. pak::lockfile_install(".github/pkg.lock")
  2. pak:::remote(function(...) { …
  3. err$throw(res$error)
  ---
  Subprocess backtrace:
   1. base::withCallingHandlers(cli_message = function(msg) { …
   2. get("lockfile_install_internal", asNamespace("pak"))(...)
   3. plan$install_sysreqs()
   4. pkgdepends:::sysreqs_install(srq$result %||% srq, config)
   5. base::lapply(cmds, function(cmd) { …
   6. local FUN(X[[i]], ...)
   7. processx::run(sh, cmdline, stdout_callback = callback, stderr_to_stdout = …
   8. base::throw(new_process_error(res, call = sys.call(), echo = echo, …
   9. | base::signalCondition(cond)
  10. global (function (e) …
  Execution halted
  Error: Process completed with exit code 1.
```

--- a/.github/workflows/check-release.yaml
+++ b/.github/workflows/check-release.yaml
@@ -47,7 +47,8 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: rcmdcheck, DBI=?ignore, odbc=?ignore
+          extra-packages: rcmdcheck
+          # extra-packages: rcmdcheck, DBI=?ignore, odbc=?ignore
 
       - uses: r-lib/actions/check-r-package@v2
         # with:

--- a/cran-comments.md
+++ b/cran-comments.md
@@ -11,8 +11,8 @@ Test environments
 1. Local Ubuntu, R 4.2.2
 2. Local Win10, R 4.2.3 patched
 3. R-hub
-    1. [Ubuntu Linux 20.04 LTS, R-release, GCC](https://builder.r-hub.io/status/REDCapR_1.1.0.tar.gz-65720cf9ae694996b81294e2ade39175)
-    2. [Fedora Linux, R-devel, clang, gfortran](https://builder.r-hub.io/status/REDCapR_1.1.0.tar.gz-dfbf2b851e0c48aeaaf8e85c9c34c332)
+    1. [Ubuntu Linux 20.04 LTS, R-release, GCC](https://builder.r-hub.io/status/REDCapR_1.1.9005.tar.gz-aa1d5ab8d07d453db1f07927d1ad23f4)
+    2. [Fedora Linux, R-devel, clang, gfortran](https://builder.r-hub.io/status/REDCapR_1.1.9005.tar.gz-a56bc6477bbb48d2a4aa5d1872481dac)
     3. [Windows Server](https://builder.r-hub.io/status/REDCapR_1.1.9005.tar.gz-34917c524b674b82860b920d177844f1)
 4. [win-builder](https://win-builder.r-project.org/eG6x3HZ9ITqN), development version.
 5. [GiHub Actions](https://github.com/OuhscBbmc/REDCapR/actions), Ubuntu 20.04 LTS

